modules/nextflow/src/main/groovy/nextflow/executor/GridTaskHandler.groovy

```diff
@@ -99,21 +101,44 @@ class GridTaskHandler extends TaskHandler {
 
     protected ProcessBuilder createProcessBuilder() {
 
-        // -- log the qsub command
+        // -- create the submit command
         def cli = executor.getSubmitCommandLine(task, wrapperFile)
+
+        // -- run script is piped from stdin when fusion is enabled
+        if( fusionEnabled() ) {
+            cli.removeAll { it.contains(wrapperFile.name) }
+        }
```

Comment:
@pditommaso The fusion process builder currently removes the wrapper file from the submit command. We could also split `getSubmitCommandLine()` into two methods, one for the command without the wrapper file line and one for the wrapper file, so that this logic isn't needed.

Also, the condor executor copies the wrapper file to `.command.condor`, so we either need to handle that case here or implement my suggestion.